packages/StatusTracker/src/StatusTracker.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Point from "./components/Point";
+import { kinds } from "./types";
+import * as sc from "./StatusTracker.styles";
+
+function StatusTracker(props) {
+  const { children } = props;
```

Comment:
why not destructure one line up?

Reply:
It just in paprika we seem like always destructure inside the function

Reply:
^ yeap is a paprika thing 🙏